### DISCUSSION TITLE
New editor discussion

### DISCUSSION THREAD

**Discussion by Derkades** [NamelessMC/Nameless#1904] [Ideas]
Vote using reactions on this issue

:+1: Change the editor
:-1: Current editor is fine

Use this language for custom pages (admin stuff):
:smile: bbcode
:rocket: markdown (some editors have support for html and thus iframes)
:eyes: html

Use this language for forum posts, replies, profile posts, etc. (user stuff):
:heart: bbcode
:confused: : markdown
:tada: html

If you have suggestions for editor libraries to use, please leave a comment. They need to be compatible with the MIT license.

**Comment by MarcosBarranquero** [comment]
Personally these are the ones I see best, they have the spell checker which from my point of view is one of the most important things in a forum text editor and they look powerful.

- [TinyMCE](https://www.tiny.cloud/)
- [Quill](https://quilljs.com/)

**Comment by Derkades** [comment]
> * [TinyMCE](https://www.tiny.cloud/)

Nameless currently uses TinyMCE
